pkgs/development/python-modules/pomegranate/default.nix

```
@@ -1,19 +1,41 @@
-{ lib, buildPythonPackage, fetchFromGitHub, numpy, scipy, cython, networkx, joblib, nose, pyyaml }:
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, fetchpatch
+, numpy
+, scipy
+, cython
+, networkx
+, joblib
+, pandas
+, nose
+, pyyaml
+}:
+
 
 buildPythonPackage rec {
   pname = "pomegranate";
-  version = "0.11.2";
+  version = "0.13.5";
 
   src = fetchFromGitHub {
     repo = pname;
     owner = "jmschrei";
     rev = "v${version}";
-    sha256 = "070ciwww1lhjmfwd5n1kcwgxwbgdfvmhjs4l156bnf08z9dlrafl";
+    sha256 = "1hbxchp3daykkf1fa79a9mh34p78bygqcf1nv4qwkql3gw0pd6l7";
   };
 
+  patches = lib.versionOlder version "13.6" [
```

oh, right, sorry `lib.versionOlder version "13.6"` just returns a bool
```suggestion
  patches = lib.optionals (lib.versionOlder version "13.6") [
```